content/working_with/manager/ldap-integration.md

```diff
@@ -6,56 +6,177 @@ weight: 1450
+  - sn (optional)- The surname of the user- this may be blank. If not blank, it will be stored as part of the user details on the manager.
+  - email (optional)- The e-mail address of the user- this may be blank. If not blank, it will be stored as part of the user details on the manager.
+  - memberOf (required)- If the user is a member of no groups other than their primary group (usually Domain Users) then they will be effectively unusable with {{< param product_name >}}, so this should be populated with the group membership.
+* Groups are looked up by consulting the memberOf attribute on a user (or, with nested lookups, on the groups).
+
+For other directories (e.g. openldap):
+* Base DN for lookups is the domain, split on '.', as a series of dc LDAP entities- e.g. dc=local,dc=example
+* Users bind as uid={username},ou=users,{base_dn}- e.g. uid=jbloggs,ou=users,dc=local,dc=example
+* Accounts are located by searching with the query (uid={username})- e.g. (uid=jbloggs)
+* Groups which the user is a member of are located by searching with the query (uniqueMember={object_dn})- e.g. (uniqueMember=uid=jbloggs,ou=users,dc=local,dc=example)
+* LDAP objects are expected to have the following attributes:
+  - uid (required)- The username of the user, e.g. jbloggs
+  - givenName (optional)- The given name of the user- this may be blank. If not blank, it will be stored as part of the user details on the manager.
+  - sn (optional)- The surname of the user- this may be blank. If not blank, it will be stored as part of the user details on the manager.
+  - email (optional)- The e-mail address of the user- this may be blank. If not blank, it will be stored as part of the user details on the manager.
+  - memberOf (optional)- If groups are present on user objects using this attribute they must also be on group objects if using nested lookups.
+* Groups are looked up using either the memberOf attribute (if present), or the group membership lookup query above.
+
+To override the above options, please consult the ldap set command options below.
+
+**Variables**
+
+Some of the options can accept variables which will be substituted. Such variables are provided enclosed in braces, e.g. "{base_dn}".
+The available variables for each option will be listed in the options below, and all come from the following list:
+* {base_dn}- The base DN for all LDAP searches. By default this is the same as the domain DN.
+* {domain_dn}- The domain name converted to LDAP format, e.g. local.example becomes dc=local,dc=example.
+* {object_dn}- The DN of the object (user or group) that is being checked for membership of groups.
+* {username}- The username of the user attempting to bind (for bind format) or the user attempting to log on (for all lookups).
+* {domain}- The raw domain, e.g. local.example.
 
 **Usage**
 
-```cfy ldap set [OPTIONS]```
+`cfy ldap set [OPTIONS]`
 
 **Options**
```

Comment:
The layout of this section is broken. add bullets/newlines as it all appears in one bulk.

Reply:
I've updated that section to valid markdown and tidied (or maybe messed!) it up a bit.